python/mxnet/contrib/onnx/_export/export_helper.py

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""export helper functions"""
+# coding: utf-8
+import os
+import logging
+import mxnet as mx
+
+def load_module(sym_filepath, params_filepath, input_shape):
```

Comment:
`sym.load` and `nd.load` works to get model and params objects from files but if the model is trained using old version of mxnet, it wont upgrade the model. There will is a compatibility issue. 
for example, some models has "param" or "attr" instead of "attrs" in json file.

Reply:
Nice corner case that is hard to think of 👍